core/domain/feedback_services.py

```diff
@@ -366,6 +366,34 @@ def _get_thread_from_model(thread_model):
         thread_model.created_on, thread_model.last_updated)
 
 
+def get_messages_for_exp_id(exp_id, limit=None):
```

Comment:
But is it likely that multiple feedback messages from the same thread will give the user more info about what's going on than just one message? Normally one thread would be about one issue (like on github).